eng/Versions.props

```diff
@@ -25,7 +25,8 @@
     <NewtonsoftJsonPackageVersion>13.0.1</NewtonsoftJsonPackageVersion>
     <NewtonsoftJsonVersion>$(NewtonsoftJsonPackageVersion)</NewtonsoftJsonVersion>
     <SystemDiagnosticsFileVersionInfoVersion>4.0.0</SystemDiagnosticsFileVersionInfoVersion>
-    <SystemReflectionMetadataVersion>5.0.0</SystemReflectionMetadataVersion>
+    <SystemReflectionMetadataVersion>6.0.0</SystemReflectionMetadataVersion>
```

Comment:
@marcpopMSFT, is this bump necessary?

Reply:
CC @rainersigwald who included that in the change. I assume there is a dependency between the two that required udpating both.

Reply:
My initial commit https://github.com/dotnet/sdk/commit/2ecd6a7fea7d33adb0f9da2901351d8823dade94 didn't have it, I think this must have come in a cherry-pick merge conflict resolution at some point.

Reply:
weird, I don't recall seeing merge conflicts. Did I have local uncommitted changes maybe that I forgot about?